src/gmt_init.c

```diff
@@ -12285,6 +12285,7 @@ struct GMT_CTRL *gmt_init_module (struct GMTAPI_CTRL *API, const char *lib_name,
 
 	bool is_PS;
 	unsigned int srtm_res = 0;
+	char *required = (char *)in_required;
```

Comment:
Let's hope this does not screw on VS (it normally only accepts declaration = setting in last declaration line)